.github/workflows/dbt_slim_ci.yml

```diff
@@ -68,6 +68,15 @@ jobs:
       - name: dbt test initial model(s)
         run: "dbt test $PROFILE --select state:new state:modified --exclude tag:prod_exclude tag:remove --defer --state . --project-dir ${PROJECT_DIR}"
 
+      - name: dbt test dex trades specific model(s)
+        run: |
+          if git diff --name-only ${{ github.event.pull_request.base.sha }}..${{ github.sha }} | grep -q "models/_sector/dex/trades/"; then
+            dbt run --models dex_base_trades
+            dbt run --models dex_trades
+            dbt test --models test_check_null_amount_usd
+          fi
+        working-directory: ${{env.PROJECT_DIR}}
```

Comment:
cc. @jeff-dude 
This part may be the tricky part.
I am not 100% sure, but it may represent the rough workflow in my head at the moment.